src/components/organisms/InfoTab/InfoTab.scss

```diff
@@ -0,0 +1,109 @@
+@import "scss/constants";
+
+$InfoTab-position-top: 188px;
+$InfoTab-width: 70px;
+$InfoTab-width-expanded: 464px;
+$InfoTab-body-margin-top: 56px;
+
+$InfoTab-logo-width: 50px;
+$InfoTab-logo-height: $InfoTab-logo-width;
+$InfoTab-logo-width-expanded: 124px;
+$InfoTab-logo-height-expanded: $InfoTab-logo-width-expanded;
```

Comment:
Following BEM convention when most if not all the rest of the `$scss-constants` start with a lower case.

Reply:
@goran-peoski-work fixed in 859a215